funding/manager.go

```diff
@@ -3729,12 +3729,13 @@ func (f *Manager) newChanAnnouncement(localPubKey,
 	}
 
 	// The caller of newChanAnnouncement is expected to provide the initial
-	// forwarding policy to be announced. We abort the channel announcement
-	// if they are not provided.
+	// forwarding policy to be announced. If no persisted initial policy
+	// values are found, then we will use the default policy values in the
+	// channel announcement.
 	storedFwdingPolicy, err := f.getInitialFwdingPolicy(chanID)
 	if err != nil {
-		return nil, errors.Errorf("unable to generate channel "+
-			"update announcement: %v", err)
+		log.Errorf("No initial forwarding policy found for "+
```

Comment:
I think we need to do a more strict check against `ErrChannelNotFound` to make sure it's a not found error, tho that's the only error that could ever be returned from `getInitialFwdingPolicy`.

Reply:
updated!

Reply:
ok gotcha - I understand now. 

The tricky part is passing the Default Forwarding Policy into the migration... gotta do some weird config passing again... will try 👍